client/cl_eventhandlers.lua

```diff
@@ -9,6 +10,58 @@ local function isPlayerAWitness(witnesses)
     return false
 end
 
+--hunting zones
+CreateThread(function()
+    for _, hunting in pairs(Config.Locations["hunting"]) do
+        if Config.Debug then
+            local huntingzone = CircleZone:Create(vector3(hunting.coords.x, hunting.coords.y, hunting.coords.z), hunting.radius, {
+                name = Config.Locations["hunting"].label,
+                useZ = true,
+                debugPoly = true
+            })
+
+            huntingzone:onPlayerInOut(function (isPointInside)
+                if isPointInside then
+                    QBCore.Functions.Notify("DEBUG: INSIDE HUNTING AREA: "..hunting.label, "success")
+                    print("inside")
+                else
+                    QBCore.Functions.Notify("DEBUG: OUTSIDE HUNTING AREA: "..hunting.label, "success")
+                    print("outside")
+                end
+
+                inHuntingZone = isPointInside
+            end)
+        else
+            local huntingzone = CircleZone:Create(vector3(hunting.coords.x, hunting.coords.y, hunting.coords.z), hunting.radius, {
+                name = Config.Locations["hunting"].label,
+                useZ = true,
+                debugPoly = false
+            })
+
+            huntingzone:onPlayerInOut(function (isPointInside)
+                inHuntingZone = isPointInside
+            end)
+        end
+    end
+end)
+
+CreateThread(function()
+    for _, hunting in pairs(Config.Locations["hunting"]) do
+        local blip = AddBlipForCoord(hunting.coords.x, hunting.coords.y, hunting.coords.z)
+        local huntingradius = AddBlipForRadius(hunting.coords.x, hunting.coords.y, hunting.coords.z, hunting.radius)
+        SetBlipSprite(blip, 442)
+        SetBlipAsShortRange(blip, true)
+        SetBlipScale(blip, 0.8)
+        SetBlipColour(blip, 0)
+        SetBlipColour(huntingradius, 0)
+        SetBlipAlpha(huntingradius, 40)
+        BeginTextCommandSetBlipName("STRING")
+        AddTextComponentString(hunting.label)
+        EndTextCommandSetBlipName(blip)
+    end
+end)
+---------------------------------------------------------------------------------------
```

Comment:
Not needed, please remove

Reply:
What should i remove? The Blip stuff all together or the lines from 60 to 63 ?